src/main/java/screens/login_registration/FileUser.java

```diff
@@ -75,13 +94,4 @@ public Map<String, UserRegSaveRequest> getAccounts() {
         return this.accounts;
```

Comment:
For my enrolment use case, I use this method to access the students in the map and then search for the desired student. To do so I would need this to be a static method:
<img width="867" alt="Screen Shot 2022-12-02 at 2 42 47 PM" src="https://user-images.githubusercontent.com/84947592/205373450-e7c251c8-ed76-49cd-ba81-8d03256d9ef0.png">

Reply:
You can call FileUser from your interactor bc of the dependency inversion--FIleUser implements UserGateway and LoginGateway and LogoutGateway which are in the use case layer